### PR TITLE
[ADD] module 'account_invoice_analysis_analytic'

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
# Invoices Analysis by analytic account

This module is meant to extend the report 'Invoices Analysis' adding the
analytic account
# Usage

Go to 'Reporting | Accounting | Invoices Analysis'. It is possible to
group by or filter by analytic account.

### CODE REVIEW

File: account_invoice_analysis_analytic/report/account_invoice_report.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp.osv import fields, orm
```
Comment (amoya-dx) [inline account_invoice_analysis_analytic/report/account_invoice_report.py]: It's better to use new API

File: account_invoice_analysis_analytic/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+Go to 'Reporting | Accounting | Invoices Analysis'. It is possible to
+group by or filter by analytic account.
+
+
+Known issues / Roadmap
+======================
+
+Credits
+=======
+
```
Comment (angelmoya) [inline account_invoice_analysis_analytic/__openerp__.py:50]: ADD Bug tracker like in https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

**Comment by angelmoya** [comment]
:+1:  Tested on runbot, functional OK.

**Comment by JordiBForgeFlow** [comment]
Added an additional module 'account_invoice_analysis_inv_number'.

# Invoices Analysis by invoice number

This module is meant to extend the report 'Invoices Analysis' adding the
invoice internal number, the supplier invoice number and the journal entry.
This is useful to allow the user to drill down.

**Comment by feketemihai** [comment]
@jbeficent Do you still need this one, since is on v7?

**Comment by pedrobaeza** [comment]
Closing as no answer. Feel free to reopen if needed.

**Pull request closed without merging**